client/src/pages/Home/FooterNav/FooterNav.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { GrMail } from "react-icons/gr";
+import "./FooterNav.css";
+
```

Comment:
Please fix indentation of this file

Reply:
Thanks Ewa! Can I ask which lines should be fixed, please? Is it about line 26? Is it totally unbalanced?
<img width="1440" alt="Screenshot 2022-01-29 at 23 50 03" src="https://user-images.githubusercontent.com/82678328/151681423-c05cb1f4-d773-41ff-9cd8-94eed1ee704a.png">

Reply:
I used "command" + "K" + "D" to format my code in VSC. Do you think this is reasonable?